app/views/calls/_note.html.erb

```diff
@@ -1,3 +1,4 @@
-<div class="list-item">
-  <b><%= note.author.first_name %>:</b>&nbsp;<%= note.body %>
+<div>
+  <%= simple_format(note.author.first_name, class: 'note-author') %>
```

Comment:
We don't need `simple_format` for the author's name because we're not trying to transform a string into HTML. The method wraps the given string in `<p>` tags, so I think this can be:
```suggestion
  <p class="note-author"><%= note.author.first_name %></p>
```